docs/proposals/sharding-by-partitions.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Sharding timeseries by partitions"
+linkTitle: "Sharding timeseries by partitions"
+weight: 1
+slug: shard-by-partitions
+---
+
+- Author: [Harry John](https://github.com/harry671003)
+- Date: May 2023
+- Status: Proposed
+---
+
+## Introduction
+
+Cortex currently supports two sharding strategies - default and shuffle sharding. 
+Shuffle sharding was introduced with the goal of reducing the blast radius and to provide better tenant isolation. However, the current implementation of shuffle sharding in Cortex cannot tolerate any two nodes in the cluster going down at the same time.
```

Comment:
s/any two nodes in the cluster going down at the same time/2 nodes in different AZs for a given tenant shard/g